intel_extension_for_transformers/neural_chat/server/config/neuralchat.yaml

```diff
@@ -56,23 +56,23 @@ tts_chinese:
         output_audio_path: "./output_audio.wav"
 
 retrieval:
-    enable: true
+    enable: false
     args:
         retrieval_type: "dense"
         input_path: "../../assets/docs/"
         embedding_model: "hkunlp/instructor-large"
         persist_dir: "./output"
         max_length: 512
-        process: true
+        process: false
 
 cache:
-    enable: true
+    enable: false
     args:
         config_dir: "../../pipeline/plugins/caching/cache_config.yaml"
         embedding_model_dir: "hkunlp/instructor-large"
 
 safety_checker:
-    enable: true
+    enable: false
 
 # task choices = ['textchat', 'voicechat', 'retrieval', 'text2image', 'finetune']
-tasks_list: ['textchat', 'retrieval']
+tasks_list: ['textchat', 'voicechat', 'finetune']
```

Comment:
'textchat', 'voicechat', 'finetune' can't be put together.

Reply:
change back to `tasks_list: ['textchat', 'retrieval']`